src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -0,0 +1,364 @@
+                    }
+                });
+            },
+            isItemDrilledIn(id) {
+                return this.drilledIn === id;
+            },
+            updatePosition(item, newPosition) {
+                let newStyle = this.convertPosition(newPosition);
+                item.config.rawPosition = newPosition;
+                item.style = newStyle;
+            },
+            bypassSelection($event) {
+                if (this.dragInProgress) {
+                    if ($event) {
+                        $event.stopImmediatePropagation();
+                    }
+                    return;
+                }
+            },
+            endDrag(item) {
+                this.dragInProgress = true;
+                setTimeout(function () {
+                    this.dragInProgress = false;
+                }.bind(this), 0);
+
+                item.config.mutatePosition();
+            },
+            mutate(path, value) {
+                this.openmct.objects.mutate(this.newDomainObject, path, value);
+            },
+            handleDrop($event) {
+                $event.preventDefault();
+
+                let domainObject = JSON.parse($event.dataTransfer.getData('domainObject'));
+                let elementRect = this.$el.getBoundingClientRect();
+                this.droppedObjectPosition = [
+                    Math.floor(($event.pageX - elementRect.left) / this.gridSize[0]),
+                    Math.floor(($event.pageY - elementRect.top) / this.gridSize[1])
+                ];
+
+                if (this.isTelemetry(domainObject)) {
+                    this.addAlphanumeric(domainObject, this.droppedObjectPosition);
+                } else {
+                    this.checkForDuplicatePanel(domainObject);
+                }
+            },
+            checkForDuplicatePanel(domainObject) {
+                let id = this.openmct.objects.makeKeyString(domainObject.identifier);
+                let panels = this.newDomainObject.configuration.panels;
+
+                if (panels && panels[id]) {
+                    let prompt = this.openmct.overlays.dialog({
+                        iconClass: 'alert',
+                        message: "This item is already in layout and will not be added again.",
+                        buttons: [
+                            {
+                                label: 'OK',
+                                callback: function () {
+                                    prompt.dismiss();
+                                }
+                            }
+                        ]
+                    });
+                }
+            },
+            addAlphanumeric(domainObject, position) {
+                let alphanumeric = {
+                    identifier: domainObject.identifier,
+                    position: position,
+                    dimensions: DEFAULT_TELEMETRY_DIMENSIONS,
+                    displayMode: 'all',
+                    value: 'sin',
```

Comment:
@larkin I made changes to set the default value in this order:

- If there's a range value, it defaults to the first item in the array.
- If no range is defined, it defaults to the first value that is returned by `metadata.values()` that is non time-domain. 
- Otherwise, it defaults to the first value returned by `metadata.values()`.

Please let me know if anything else needs to be changed. If not, the code is ready to be merged. Thanks for the review!